### PR TITLE
make option exclude-parent-properties, other minor clean-up

### PR DESCRIPTION
**Pull request by EdColeman**
- Change command option from `exclude-parent` to `exclude-parent-properties` for clarity.
- clarify command description present to uses
- comment clean-up

From additional comment in PR #3562 made after merge.

### CODE REVIEW

File: shell/src/main/java/org/apache/accumulo/shell/commands/CreateNamespaceCommand.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
       TableNotFoundException, IOException, ClassNotFoundException, NamespaceExistsException,
       NamespaceNotFoundException {
 
-    // validate that copy config and copy properties options are mutually exclusive.
+    // exclude parent properties only valid with copy config
```
Comment (ctubbsii) [inline shell/src/main/java/org/apache/accumulo/shell/commands/CreateNamespaceCommand.java:47]:
```suggestion
    // exclude parent properties; only valid with copy config
```

File: shell/src/main/java/org/apache/accumulo/shell/commands/CreateTableCommand.java [hunk 2]
```diff
@@ -101,7 +101,7 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
           new TreeSet<>(shellState.getAccumuloClient().tableOperations().listSplits(oldTable)));
     }
 
-    // allow only copy config or copy properties,
+    // exclude parent properties only valid with copy config
```
Comment (ctubbsii) [inline shell/src/main/java/org/apache/accumulo/shell/commands/CreateTableCommand.java:104]:
```suggestion
    // exclude parent properties; only valid with copy config
```

**Review by ctubbsii — COMMENTED** [review]
Good to go after comments update

**Review by ctubbsii — APPROVED** [review]

**Comment by ctubbsii** [comment]
This PR doesn't seem to be triggering the GitHub Actions CI checks for some reason. Not really sure what's going on there.

**Comment by EdColeman** [comment]
I manually triggered a re-run just to double check (https://github.com/apache/accumulo/actions/runs/5662956604)

**Pull request merged**